dev/buildtool/container_commands.py

```diff
@@ -73,6 +73,14 @@ def init_argparser(self, parser, defaults):
             help="Show proposed actions, don't actually do them. Default True.",
         )
 
+        self.add_argument(
+            parser,
+            "tag_java11",
+            defaults,
+            "",
```

Comment:
Might be slightly easier to use if we make this a `bool` and use a constant for the list of services.

See: [SPINNAKER_RUNNABLE_REPOSITORY_NAMES](https://github.com/spinnaker/buildtool/blob/a780b26f4c78d1651f7c11c378ec9e50f77248e2/dev/buildtool/__init__.py#L6)